src/DependencyInjection/BaldinofRoadRunnerExtension.php

```diff
@@ -157,4 +163,35 @@ private function aliasIfNotExists(ContainerBuilder $container, string $alias, st
 
         $container->setAlias($alias, $id);
     }
+
+    private function registerRpcRelay(ContainerBuilder $container): void
+    {
+        $rpcDsn = parse_url((string) getenv('RR_RPC'));
+        if (!is_array($rpcDsn) || !array_key_exists('scheme', $rpcDsn)) {
+            return;
+        }
+
+        switch ($rpcDsn['scheme']) {
+            case 'tcp':
+                $container->getDefinition('baldinof_road_runner.rr.rpc_relay')->setArguments([
+                    $rpcDsn['host'],
+                    $rpcDsn['port'],
+                    SocketRelay::SOCK_TCP,
+                ]);
+                break;
+            case 'unix':
+                $soketPath = $rpcDsn['host'].$rpcDsn['path'];
+                //is path relative? Make it absolute, from project root.
+                if ('/' !== $soketPath[0]) {
+                    $soketPath = $container->getParameter('kernel.project_dir').'/'.$soketPath;
+                }
+
+                $container->getDefinition('baldinof_road_runner.rr.rpc_relay')->setArguments([
+                    $soketPath,
+                    null,
+                    SocketRelay::SOCK_UNIX,
+                ]);
+                break;
```

Comment:
An exception could be thrown in case of unsupported scheme.

Reply:
Fixed.